model/k8s/metrics.yaml

```
@@ -318,6 +318,69 @@ groups:
       [`k8s.hpa`](../resource/k8s.md#horizontalpodautoscaler) resource.
     instrument: updowncounter
     unit: "{pod}"
+  - id: metric.k8s.hpa.metric.target.cpu.value
```

```suggestion
  - id: metric.k8s.hpa.spec.metric.target.cpu.value
```
Should we make a distinction between spec vs status in k8s metrics if the field reported shows up in both?

---

I assume it's not feasible to get any metrics from the hpa status. If so, I don't think we need to add `spec`. It'd be a precedent that can cause significant complications to all other metric names :)

---

The hpa reports the current metric in status, but if we want to report that in the future we can use currentmetric to differentiate with target. You are right, best to not complicate this further.